metricbeat/beater/event.go

```diff
@@ -51,9 +47,7 @@ func (b eventBuilder) build() (common.MapStr, error) {
 		"type":       typeName,
 
 		common.EventMetadataKey: b.metadata,
-		b.moduleName: common.MapStr{
-			b.metricSetName: event,
-		},
+		b.moduleName:            event,
```

Comment:
Having the namespace of module and metricset here is critical to prevent namespace conflicts. This guarantees to no metricset can conflict with an other. In addition we move the "burden" of doing the namespacing correctly to the person implementing the metricset. -1 on doing this change.